src/main/resources/META-INF/plugin.xml

```diff
@@ -347,11 +347,7 @@
         <javaCodeActionParticipant kind="quickfix"
                                    group="jakarta"
                                    targetDiagnostic="jakarta-cdi#RemoveProducesOrInject"
-                                   implementationClass="io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.cdi.RemoveProduceAnnotationQuickFix"/>
-        <javaCodeActionParticipant kind="quickfix"
-                                   group="jakarta"
-                                   targetDiagnostic="jakarta-cdi#RemoveProducesOrInject"
-                                   implementationClass="io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.codeAction.proposal.quickfix.RemoveInjectAnnotationQuickFix"/>
+                                   implementationClass="io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.cdi.ConflictProducesInjectQuickFix"/>
```

Comment:
@vaisakhkannan Now that I'm reviewing your PR, I have a better understanding of what you were saying during your demo this morning.  It seems that `ConflictProducesInjectQuickFix` does exactly what `RemoveProduceAnnotationQuickFix` and `RemoveInjectAnnotationQuickFix` were doing together which is why you were able to swap them here and not affect the behaviour. I think this means we have redundant code and could possibly delete one of these implementations. It would be helpful to review LSP4Jakarta (https://github.com/eclipse/lsp4jakarta) to understand if it has its own `ConflictProducesInjectQuickFix` class, if it is actually using it and if not perhaps we should remove it from both repositories.

Reply:
@mrglavas I reviewed the code for LSP4Jakarta (https://github.com/eclipse/lsp4jakarta). I noticed that they are using the [RemoveProducesAndInjectAnnotationsQuickFix](https://github.com/eclipse/lsp4jakarta/blob/main/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/internal/cdi/RemoveProducesAndInjectAnnotationsQuickFix.java), [RemoveInjectAnnotationQuickFix](https://github.com/eclipse/lsp4jakarta/blob/main/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/internal/cdi/RemoveInjectAnnotationQuickFix.java), and [RemoveProduceAnnotationQuickFix](https://github.com/eclipse/lsp4jakarta/blob/main/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/internal/cdi/RemoveProduceAnnotationQuickFix.java).

The way we are implementing `ConflictProducesInjectQuickFix` for the target diagnostic is through `"jakarta-cdi#RemoveProducesOrInject"`. I checked the test; it matches.

The way you mentioned is correct. 
`It seems that ConflictProducesInjectQuickFix does exactly what RemoveProduceAnnotationQuickFix and RemoveInjectAnnotationQuickFix were doing together which is why you were able to swap them here and not affect the behaviour.`

Yes, I hope that we can remove ConflictProducesInjectQuickFix from both repositories. Instead, we can utilize RemoveProduceAnnotationQuickFix and RemoveInjectAnnotationQuickFix together.

Reply:
@vaisakhkannan In lsp4Jakarta there is no ConflictProducesInjectQuickFix class and if we remove ConflictProducesInjectQuickFix from our project, it won't affect any other quick fixes?

Reply:
> @mrglavas I reviewed the code for LSP4Jakarta (https://github.com/eclipse/lsp4jakarta). I noticed that they are using the [RemoveProducesAndInjectAnnotationsQuickFix](https://github.com/eclipse/lsp4jakarta/blob/main/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/internal/cdi/RemoveProducesAndInjectAnnotationsQuickFix.java), [RemoveInjectAnnotationQuickFix](https://github.com/eclipse/lsp4jakarta/blob/main/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/internal/cdi/RemoveInjectAnnotationQuickFix.java), and [RemoveProduceAnnotationQuickFix](https://github.com/eclipse/lsp4jakarta/blob/main/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/internal/cdi/RemoveProduceAnnotationQuickFix.java).
> 
> They are using [RemoveProducesAndInjectAnnotationsQuickFix](https://github.com/eclipse/lsp4jakarta/blob/main/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/internal/cdi/RemoveProducesAndInjectAnnotationsQuickFix.java) in the corresponding codeAction
> 
> [https://github.com/eclipse/lsp4jakarta/blob/2ca7a71c9127f27358af208f0b2650d6e9caa3e2/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/plugin.xml#L286#291](https://github.com/eclipse/lsp4jakarta/blob/2ca7a71c9127f27358af208f0b2650d6e9caa3e2/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/plugin.xml#L286%23291)
> 
> The way we are implementing `ConflictProducesInjectQuickFix` for the target diagnostic is through `"jakarta-cdi#RemoveProducesOrInject"`. I checked the test; it matches.
> 
> The way you mentioned is correct. `It seems that ConflictProducesInjectQuickFix does exactly what RemoveProduceAnnotationQuickFix and RemoveInjectAnnotationQuickFix were doing together which is why you were able to swap them here and not affect the behaviour.`
> 
> Yes, I hope that we can remove `ConflictProducesInjectQuickFix` from both repositories. Instead, we can utilize `RemoveProduceAnnotationQuickFix` and `RemoveInjectAnnotationQuickFix` together.

@mrglavas , Can you confirm this as well, Michael?